lldb/cmake/modules/AddLLDB.cmake

```diff
@@ -151,6 +151,36 @@ function(add_lldb_library name)
   endif()
 endfunction(add_lldb_library)
 
+# BEGIN Swift Mods
+function(add_properties_for_libswift target)
+  if (LIBSWIFT_BUILD_MODE)
+    if (CMAKE_SYSTEM_NAME MATCHES "Darwin")
+      if(LIBSWIFT_BUILD_MODE MATCHES "HOSTTOOLS|.*HOSTLIBS")
+        target_link_directories(${target} PRIVATE
+            "${CMAKE_OSX_SYSROOT}/usr/lib/swift"
```

Comment:
This line looks suspicious to me but I also don't fully understand what it does. Does this end up being in an rpath pointing into the build machine's SDK?

Reply:
For macOS: only in "bootstrapping" mode, which is only used for testing. For other modes the rpath is pointing to `/usr/lib/swift`.

For Linux: yes, but I couldn't figure out how to fix that. But this is already the case without this change.